src/shared/styles/styledUtils.ts

```diff
@@ -0,0 +1,12 @@
+const letterSpacingValue = (fontSize: string | number, letterSpacingPercentage: number) => {
+  const numericFontSize = typeof fontSize === 'string' ? parseFloat(fontSize) : fontSize;
+
+  if (Number.isNaN(numericFontSize)) {
+    console.error('[letterSpacingValue] Invalid fontSize value:', fontSize);
+    return 'normal';
+  }
+
+  return `${(numericFontSize * letterSpacingPercentage) / 100}px`;
+};
+
+export { letterSpacingValue };
```

Comment:
```suggestion
export const letterSpacingValue = (fontSize: string | number, letterSpacingPercentage: number) => {
  const numericFontSize = typeof fontSize === 'string' ? parseFloat(fontSize) : fontSize;

  if (Number.isNaN(numericFontSize)) {
    console.error('[letterSpacingValue] Invalid fontSize value:', fontSize);
    return 'normal';
  }

  return `${(numericFontSize * letterSpacingPercentage) / 100}px`;
};
```
두 번씩 선언하는 것 보단, 이렇게 한 번에 export 해주면 좋지 않을까 싶네요!

Reply:
> 두 번씩 선언하는 것 보단, 이렇게 한 번에 export 해주면 좋지 않을까 싶네요!

개발하면서는 styleUtils에 다른 함수도 생길 여지가 있다는 점에서 export를 분리해서 했어요.
지금보니 하나의 함수만 정의하고 있네요. 한번에 선언해주는게 좋아보여요!